sources/packages/web/src/composables/useFormioComponentLoader.ts

```diff
@@ -0,0 +1,44 @@
+import { EducationProgramService } from "../services/EducationProgramService";
+import { EducationProgramOfferingService } from "@/services/EducationProgramOfferingService";
+import { useFormioUtils } from ".";
+
+/**
+ * Common methods to load components data on Form.IO that could
+ * be reusable or at least simplify the form data load logic.
+ * @returns Methods to help loading the data into Form.IO forms.
+ */
+export function useFormioComponentLoader() {
+  const formioUtils = useFormioUtils();
+  // get offering date of the selected offering and set to the hidden field (selectedOfferingDate) in formio
```

Comment:
As agreed, please start the sentence with a capital letter and end it with a period.